{{cookiecutter.project_slug}}/compose/local/django/celery/worker/start

```diff
@@ -4,4 +4,4 @@ set -o errexit
 set -o nounset
 
 
-celery -A config.celery_app worker -l INFO
+watchmedo auto-restart --directory=./ --pattern="*tasks.py;*celery_app.py" --recursive -- celery -A config.celery_app worker -l INFO
```

Comment:
Looks like we can do it with `watchgod` which is already installed if we use async:

```suggestion
watchgod celery.__main__.main --args -A config.celery_app beat -l INFO
```

Reply:
`beat` has to be `worker` I guess?

Reply:
Yes not sure why I suggested beat, that's definitely wrong